website/versioned_docs/version-7.1.0/preset-env.md

```diff
@@ -174,10 +174,12 @@ Enable ["loose" transformations](http://2ality.com/2015/12/babel6-loose-mode.htm
 
 ### `modules`
 
-`"amd" | "umd" | "systemjs" | "commonjs" | "cjs" | false`, defaults to `"commonjs"`.
+`"amd" | "umd" | "systemjs" | "commonjs" | "cjs" | "auto" | false`, defaults to `"auto"`.
 
 Enable transformation of ES6 module syntax to another module type.
 
+The default `auto` will automatically select `false` if ES6 module syntax is already supported by the caller, or `"commonjs"` otherwise.
```

Comment:
Maybe we could be more explicit? `by the caller (like babel-loader or rollup-plugin-babel)`

Reply:
Would it suffice to add exactly that? I'm not too familiar with the implementation of the option